prime-router/metadata/fhir_mapping/hl7/datatype/XONToOrganization.yml

```diff
@@ -0,0 +1,80 @@
+resourceType: Organization
+
+id:
+  type: STRING
+  valueOf: "GeneralUtils.generateResourceId()"
+  expressionType: JEXL
+
+name:
+  type: STRING
+  valueOf: XON.1
+  expressionType: HL7Spec
+
+extension_1-nameType:
+  generateList: true
+  expressionType: nested
+  expressionsMap:
+    url:
+      type: SYSTEM_URL
+      value: organization-name-type
+    valueCodeableConcept:
+      valueOf: datatype/CodeableConcept
+      generateList: true
+      expressionType: resource
+      specs: XON.2
+
+identifier_1:
+  condition: $xon10 NULL
+  valueOf: datatype/Identifier_var
```

Comment:
Big picture I agree that we definitely need to clean up how we map `Identifier` across the board, but I don't think I've seen enough of the inventory to truly know that XON.3-10 map to an Identifier uniquely or there are other cases where a different HL7 datatype will map similarly.  I can put some more thought into it.

WRT deleting files.  I'm overall trying to avoid deleting anything for now and we can hopefully get consensus on the directory structure so we'll know where the "cleaned" schemas are.